src/ResourceManager/Common/Commands.Common.Strategies/NestedResourceConfig.cs

```diff
@@ -36,7 +36,7 @@ public sealed class NestedResourceConfig<TModel, TParenModel> : IEntityConfig<TM
         /// </summary>
         public IEntityConfig<TParenModel> Parent { get; }
 
-        public Func<TModel> CreateModel { get; }
+        public Func<string, TModel> CreateModel { get; }
```

Comment:
This is a breaking API change, so we will need to update the Assembly version

Reply:
Already updated to 2.0 in @sergey-shandar merged PR https://github.com/Azure/azure-powershell/pull/5129